cmd/kyverno/main.go

```diff
@@ -515,6 +517,14 @@ func main() {
 			setup.KyvernoClient,
 			backgroundServiceAccountName,
 		)
+		ephrs, err := StartAdmissionReportsCounter(signalCtx, setup.MetadataClient)
+		if err != nil {
+			setup.Logger.Error(errors.New("failed to start admission reports watcher"), "failed to start admission reports watcher")
```

Comment:
Do we need to print out same message twice?

Reply:
Can i fix it in a follow up ?

Reply:
It's up to you.

Reply:
👍

Reply:
Oh yeah, this one is fine.